modules/imaging_browser/php/module.class.inc

```diff
@@ -102,7 +84,42 @@ class Module extends \Module
                     "new-scans"
                 )
             ];
+        case 'candidate':
+            $candidate = $options['candidate'];
+            if ($candidate === null) {
+                return [];
+            }
+
+            $scansummary = $DB->pselect(
+                "SELECT s.Visit_label,
+                    mst.Scan_type,
+                    COALESCE(fqc.QCStatus, '') as QC,
+                    COUNT(*) as nfiles
+                 FROM files f JOIN session s ON (s.ID=f.sessionID)
+                    JOIN mri_scan_type mst ON (f.AcquisitionProtocolID=mst.ID)
+                    LEFT JOIN files_qcstatus fqc ON (fqc.FileID=f.FileID)
+                WHERE CandID=:cid
```

Comment:
should limit the query to f.OuputType='native' for now. All other output types will be very project dependant while 'native' refers to the raw data only

Reply:
I've restricted it to 'native' and it seems to restrict it enough to re-add "Other"